src/components/canrawview/gui/crvgui.h

```diff
@@ -5,6 +5,12 @@
 #include "ui_canrawview.h"
 #include <memory>
 
+enum class ColName { rowID = 0, time, id, dir, dlc, data };
+
+enum class Type { uint = 0, hex, doubl, str };
```

Comment:
- Create header file instead of defining one enum in two places. 
- Use some more descriptive names as CRVColumnType. Enum names should start at least from upper case. 
- doubl -> double
- use coma after last element to trick clang-format and put each enum elem in new line.

Reply:
Double can't be used as it's a key word. I set names to double_type and so on.